js/geonovum/leafletfigures.js

```diff
@@ -0,0 +1,40 @@
+define(
+  ["geonovum/deps/leaflet",],
+  function(L){
+  /**
+   * Makes figures scalable via zoom and pan function
+   *
+   */
+  "use strict";
+  return {
+    run: function(conf, doc, cb) {
+      Array
+        .from(doc.querySelectorAll("figure.scalable img"))
+        .forEach(function(image) { 
+          var container = image.parentElement;
+          var div = document.createElement("div");
+          div.id = "leaflet-figure";
+          container.insertBefore(div, image);
```

Comment:
I'm wondering if it might be better to do:

```JS
container.replaceChild(div, image);
```

Then you don't need to worry about removing the image at the end.